azure-kusto-data/tests/test_helpers.py

```diff
@@ -11,28 +12,51 @@
 import numpy
 
 
-def test_dataframe_from_result_table():
-    """Test conversion of KustoResultTable to pandas.DataFrame, including fixes for certain column types"""
+class PandasTests(unittest.TestCase):
+    def test_dataframe_from_result_table(self):
+        """Test conversion of KustoResultTable to pandas.DataFrame, including fixes for certain column types"""
 
     with open(os.path.join(os.path.dirname(__file__), "input", "dataframe.json"), "r") as response_file:
         data = response_file.read()
 
     response = KustoResponseDataSetV2(json.loads(data))
-    df = dataframe_from_result_table(response.primary_results[0])
+    # Test that receives both types of dictionary parameters. Test that type conversion doesn't override column name conversion
+    test_dict_by_name = {"RecordName": lambda col, frame: frame[col].astype("str"), "RecordInt64": lambda col, frame: frame[col].astype("int64")}
+    test_dict_by_type = {"int": lambda col, frame: frame[col].astype("int32")}
+    test_int_to_float = {"int": "float64"}
+    df = dataframe_from_result_table(response.primary_results[0], converters_by_type=test_dict_by_type, converters_by_column_name=test_dict_by_name)
+    df_int_to_float = dataframe_from_result_table(response.primary_results[0], converters_by_type=test_int_to_float)
```

Comment:
Move df_int_to_float initialization to right before where it used (also test_int_to_float)